### ISSUE TITLE
[Bug]: Error Boundaries Caveats are not added in the new docs

### ISSUE THREAD

**Issue by robertwt7** [reactjs/react.dev#6991]
### Summary

In the [previous docs](https://legacy.reactjs.org/docs/error-boundaries.html), error boundaries caveats are clearly defined. In the [new one](https://react.dev/reference/react/Component#catching-rendering-errors-with-an-error-boundary) however, it's not.

If we want to direct all new react eng to the new docs we should be adding this details to the new one as well

> Error boundaries do not catch errors for:
Event handlers ([learn more](https://legacy.reactjs.org/docs/error-boundaries.html#how-about-event-handlers))
Asynchronous code (e.g. setTimeout or requestAnimationFrame callbacks)
Server side rendering
Errors thrown in the error boundary itself (rather than its children)

### Page

https://react.dev/reference/react/Component#catching-rendering-errors-with-an-error-boundary

**Comment by ShakedAdi** [comment]
Hi! I'm reviewing this issue and noticed that canveats about Error Boundaries apear to already be included in the current docs.

<img width="840" height="317" alt="Image" src="https://github.com/user-attachments/assets/7a4214d0-3255-4876-8a0f-377fe8a595f7" />

tell me if I misunderstood?

**Comment by robertwt7** [comment]
yeah it seems like its there now, I think when i reported it last time it wasn't there. we can close this thankyou
